lib/rex/ui/text/dispatcher_shell.rb

```diff
@@ -646,11 +641,35 @@ def unblock_command(cmd)
   # ArgumentError on unbalanced quotes return the remainder of the string as if
   # the last character were the closing quote.
   #
+  # This code was originally taken from Shellwords#shellsplit
```

Comment:
Probably worth a github link to the exact sha that this was using 👀

Reply:
Got a link to that method in the Ruby repo added in 👍